scrapinghub/client.py

```diff
@@ -630,6 +632,8 @@ def update_tags(self, add=None, remove=None):
 
             >>> job.update_tags(add=['consumed'])
         """
+        if not (add or remove):
+            raise ValueError('Please provide tags to add or remove')
```

Comment:
Ok, convinced, I'll create a ticket for server-side changes and drop it from here